### DISCUSSION TITLE
Invalid website in KV service and unable to run ResilientDB

### DISCUSSION THREAD

**Discussion by WengZixuanZLXW** [apache/incubator-resilientdb#162] [Q&A] [answered]
When I follow the instruction to run ResilientDB by command **./service/tools/kv/server_tools/start_kv_service.sh**, it try to get the package of zlib from address **https://zlib.net/zlib-1.2.12.tar.gz**. 
However, a warning is raised that **WARNING: Download from https://zlib.net/zlib-1.2.12.tar.gz failed: class java.io.FileNotFoundException GET returned 404 Not Found**. By my check, zlib website actually disabled this address. Then a fatel error occured that **ERROR: /root/.cache/bazel/_bazel_root/4c0a3e58e6c2d936af1e2fe326a445bd/external/net_zlib_zlib/BUILD.bazel:25:11: Compiling gzlib.c failed**, which is also related with the zlib.
Because the above error, I can't run ResilientDB by the instruction. I wonder does the invalid website https://zlib.net/zlib-1.2.12.tar.gz cause the problem.

**Comment by cjcchen** [comment, marked as answer]
Thanks for letting us know.

There are two URLs there:

https://zlib.net/zlib-1.2.12.tar.gz 
https://storage.googleapis.com/bazel-mirror/zlib.net/zlib-1.2.12.tar.gz

The second one is still working.

However, I have removed the unhealthy one. 

Thanks a lot.
